testsuite/features/profiles/cloud_aws/Docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ip-172-16-1-30.eu-central-1.compute.internal/opensuse/leap:15.4
+MAINTAINER Admin User "noemail@example.com"
```

Comment:
this is deprecated according to the [docker doc](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated), use something like LABEL org.opencontainers.image.authors="SvenDowideit@home.org.au".
Also use consistently the same email and name.

Reply:
This is generated and such changes should be made in the template which did not change.
I think this should go into a separate PR